src/datasources/accounts/address-books/address-books.datasource.ts

```diff
@@ -34,7 +33,7 @@ export class AddressBooksDatasource implements IAddressBooksDatasource {
       args.chainId,
     );
     const addressBookItem = {
-      id: (max(addressBook.data.map((i) => i.id)) ?? 0) + 1,
+      id: Math.max(...addressBook.data.map((i) => i.id).concat(0)) + 1,
```

Comment:
Nit: I find the new approach quite hard to understand at a glance. It’s a good example of how Lodash can improve readability, albeit at the cost of performance. However, since the new approach involves concatenating and destructuring the array, I’m not convinced it’s any faster or more performant.

Reply:
Reverted in 48e85aaaa7141bddae48903508e873be099b3bdf.